scripts/idl/test_matter_idl_parser.py

```diff
@@ -380,9 +344,9 @@ def test_cluster_event_acl(self):
 
     def test_multiple_clusters(self):
         actual = parseText("""
-            server cluster A = 1 {}
-            client cluster B = 2 {}
-            client cluster C = 3 {}
+            server cluster A = 1 { /* FIXME */ }
```

Comment:
Were you intending to do something here?

Reply:
oops ... I think this was a reminder. I will rename it to test-comment.